webview.h

```diff
@@ -1404,18 +1404,31 @@ class webview : public browser_engine {
     }
   }
 
-  void unbind(const std::string &name) {
+  void *unbind(const std::string &name) {
```

Comment:
That's correct. Previously there was no return value, so now someone using the sync binding function still doesn't need to care about the return value, since `arg` was created internally anyway. For the async binding, the returned value helps free the function. If someone is using an async bind, it returns the value provided (`nullptr`, `bool`, etc). If using a sync binding they should expect a nullptr return. This does play into your comment about error codes.

Reply:
I support the idea of letting the user retrieve the `arg` in some way but I feel uneasy about `nullptr` meaning different things and making that part of the C++ API. Ignoring the returned `arg` may even trigger a lint warning on the caller's side.

Reply:
I support the idea of letting the user retrieve the arg in some way. Ignoring the returned arg may trigger a lint warning on the caller's side. Triggering a warning would help the caller use the API correctly but in this case the significance of the arg is defined by the user.